### ISSUE TITLE
opsu! On a IOS device? A random question. (Please read)

### ISSUE THREAD

**Issue by HyperXinon** [itdelatrisu/opsu#542]
It says desktop version only, but Ian not quite sure if this still fits here.
So hi, I wanted to ask if there’s any possible way to Convert opsu! To iOS? (Especially an iPad).
Im a big fan of osu! And opsu! And just got the new iPad. I downloaded osu!lazer which I lost the interest for because of how you play. You cant singletap or doubletap (alternate) Iam a pc player but wanted to play on my new iPad to so this is a request. 

Is there like any way to convert it? Via, profiles or TestFlight? In the best case even asking ppy for permission to upload on the AppStore? You got it on the google play store so why not the AppStore? 

Or is it the conversion to iOS?

I don’t know so this is why I ask here! 

I hope iam not bothering you guys in any kind of way, I’m just interested in a possibility for iOS.

-Have a great day and Greetings from Germany. Justin S.

**Comment by UnderSampled** [comment]
Yes, actually! The Android version (https://github.com/fluddokt/opsu) is actually a port to LibGDX (https://libgdx.com/), which is cross platform and supports iOS. These projects are not actively maintained, so you will mostly be on your own, but it shouldn't be too hard to figure out what's going on. We'd of course love any bugfixes you do figure out in the process!

It could be as simple as running libGDX's configure scripts to create the iOS wrappers. The actual wrappers are kept in a different repository (https://github.com/fluddokt/FKGdxOpsuLaunchers) from the fork, brought in through git submodules, so that the folder layout didn't have to be changed for the fork.

Technically, the version on the Google Play Store is a "third party" closed source fork of that open source one, which adds the leaderboards and Play integration. You will need to add these features for the iOS version if you desire to have those capabilities.
